blockchain/state_migration.go

```diff
@@ -528,3 +531,139 @@ func (bc *BlockChain) StopWarmUp() error {
 
 	return nil
 }
+
+// StartCollectingTrieStats collects state or storage trie statistics.
+func (bc *BlockChain) StartCollectingTrieStats(contractAddr common.Address) error {
```

Comment:
How about adding one more parameter selecting a target block number?

Reply:
That might complicate the use of the API. Let's leave it as it is and change it if we need that function.